Source/GameEngineVisuals/Program.cs

```diff
@@ -204,7 +319,13 @@ public void setPixel(int iLocX, int iLocY, char cSymbol)
                 m_cmGrid[iLocY, iLocY] = cSymbol;
             }
 
-
+            //-----------------------------------------------------------------------------
+            // setPixelMap
+            //-----------------------------------------------------------------------------		
+            public void setPixelMap(MapVertex mapVertex, char cSymbol)
+            {
+                m_cmGrid[mapVertex.m_iX, mapVertex.m_iY] = cSymbol;
+            }
 
             //-----------------------------------------------------------------------------
             // sendToGlobalBuffer
```

Comment:
I think it's better to either use xml comments (/// + Enter) or multi-line comments. Xml comments provide descriptive information about a method for example and Intellisense can even extract information from the xml comment to show information in a tooltip (description, return value, param values etc).